packages/rest/src/rest.server.ts

```diff
@@ -196,6 +198,21 @@ export class RestServer extends Context implements Server, HttpServerLike {
     };
     this._expressApp.use(cors(corsOptions));
 
+    // Place the assets router here before controllers
+    this._setupRouterForStaticAssets();
```

Comment:
This can be moved to be inside the if statement below so we only instantiate this router if a static asset is being added.

Reply:
No, I have it outside of the `if` statement on purpose as we need to register the middleware slot so that `.static()` will work.

Reply:
But you are calling this function from the `.static()` method as well ... so maybe remove it from there?